mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/persistence/MantisJobStore.java

```diff
@@ -275,34 +216,28 @@ private void archiveWorkersIfAny(IMantisJobMetadata mjmd) throws IOException {
     }
 
     public void archiveWorker(IMantisWorkerMetadata worker) throws IOException {
-        if (logger.isTraceEnabled()) { logger.trace("Archiving worker index=" + worker.getWorkerIndex()); }
```

Comment:
nit: Do you want to retain the logger.trace after removing the condition? maybe replace the string concatenation with interpolation for perf reasons.

Reply:
are trace logs useful?